### ISSUE TITLE
[Suggestion] Rescanning

### ISSUE THREAD

**Issue by ghost** [omkarmoghe/Pokemap#324]
Would it be possible to have an option to rescan the area until the pokemon go servers respond so that it doesn't leave areas unscanned? I find it to be quite problematic.

**Comment by Giggitybyte** [comment]
I think that areas which could not be scanned should go into a queue to be rescanned at the end of the whole scanning cycle. Once an area in the queue is successfully scanned it'll be removed from the queue. An area in the queue has enough unsuccessful attempts would also be removed from the queue and a message in the snackbar would tell the user that not all areas could be scanned.

That's how I would go about it, but I have no experience with Java or Android apps yet so I'm probably talking out of my ass.

**Comment by ghost** [comment]
@Giggitybyte I think that your method would be less efficient than having an option to wait for the server to respond or ask the server again until it's completely done.

Your method would keep adding queues until it's completely scanned while mine would simply wait and ask again for the area which isn't scanned instead of ignoring it.

That said, I don't have experience in programming either so maybe we're both talking out of our asses.

**Comment by Giggitybyte** [comment]
Well, let's get somebody in here who would know!
@s7092910 @omkarmoghe @comann @rohanjain1

**Comment by omkarmoghe** [comment]
yeah we are currently reworking the way we scan to handle bad responses from the server and to make sure all the area gets scanned

its sort of a rework of some of the code, so hang tight for a bit

**Comment by ghost** [comment]
@omkarmoghe Alright, v0.1 will be delayed due to the recent issues, right? You guys are doing amazing work, I've never seen an app develop this fast.
